modules/web/src/main/java/com/haulmont/masquerade/components/SideMenu.java

```diff
@@ -0,0 +1,109 @@
+package com.haulmont.masquerade.components;
+
+import com.haulmont.masquerade.Conditions;
+import com.haulmont.masquerade.util.Log;
+
+import java.util.Arrays;
+
+/**
+ * SideMenu component.
+ * <br>
+ * Supported conditions:
```

Comment:
The component should also work with non-collapsible `SideMenu`. Add to JavaDoc that these conditions are supported for collapsible `SideMenu`.

Please also check that usual `SideMenu` returns `false` for `COLLAPSED` and `true` for expanded conditions